packages/block-library/src/pullquote/theme.scss

```diff
@@ -4,12 +4,22 @@
 	margin-bottom: 1.75em;
 	color: #555;
 
+	.is-dark-theme & {
+		border-top: 4px solid $light-gray-placeholder;
```

Comment:
Isn't this an internal UI variable? Shouldn't we be avoiding the use of those on front-end block styles?

Reply:
Good question. This is where the styles were defined for the editor, so adding this to account for the styles against a dark background worked here. I believe that themes need to opt-in for this though, right? Reading from: https://github.com/WordPress/gutenberg/issues/16732

>If a theme opts in, these are provided in the front-end as well, via the theme.css stylesheet.